modules/gdnative/gdnative_api.json

```diff
@@ -590,6 +590,15 @@
           ["const godot_real", "p_t"]
         ]
       },
+      {
+        "name": "godot_vector2_move_towards",
```

Comment:
Those functions have to be added in a new GDNative core extension. This is the second PR that tries adds things which would break binary compatibility, I guess I should write up a document on how to add things to GDNative

Reply:
Oh I see now. I'm guessing I should add a new Core 1.2 API version, like this?

```JSON
{
  "core": {
    "type": "CORE",
    "version": {
      "major": 1,
      "minor": 0
    },
    "next": {
      "type": "CORE",
      "version": {
        "major": 1,
        "minor": 1
      },
      "next": {
        "type": "CORE",
        "version": {
          "major": 1,
          "minor": 2
        },
        "next": null,
        "api": [
          {
            "name": "godot_vector3_move_towards",
            "return_type": "godot_vector3",
            "arguments": [
              ["const godot_vector3 *", "p_self"],
              ["const godot_vector3 *", "p_to"],
              ["const godot_real", "p_delta"]
            ]
          },
          {
            "name": "godot_vector2_move_towards",
            "return_type": "godot_vector2",
            "arguments": [
              ["const godot_vector2 *", "p_self"],
              ["const godot_vector2 *", "p_to"],
              ["const godot_real", "p_delta"]
            ]
          }
        ]
      },
      "api": [
            ..
      ]
    },
    "api": [
         ..
    ]
  },
  "extensions": [
       ..
  ]
}
]


```

Reply:
Yes, that's right!